dist/index.html

```diff
@@ -7,10 +7,10 @@
     <title>Film Guru</title>
   <link rel="icon" href="assets/favicon.png"><script defer src="main.js"></script><link href="main.css" rel="stylesheet"></head>
   <body>
-    <!-- <section class="overlay"></section> -->
+    <section class="overlay"></section>
```

Comment:
I see that you have added the overlay, I don't think it will blend well with the colors